README.md

```diff
@@ -33,31 +33,31 @@ These are some good reads that inspired us to start with this project:
 ## Contribute
 
 All contributions are welcome, you don't have to be a full blown front-end developer to contribute to Parsy. Not 
-everybody writing a pattern should be able to implement it. In many cases you will recognize patterns by spotting 
-repetition in user interfaces, solutions to common problems. That's exactly what we are looking for! Describe the 
-problem, describe how it is 
-solved, and create a proposal with our [template]. It doesn't have to be a final version. We are here to help! Every 
-proposal will be reviewed and shaped until it is ready for approval. We believe that first time right doesn't exist, 
-or it is just a lucky shot. Even if you are not sure if the problem you described is a pattern, please submit your 
-idea that will help others to write even better proposals.
+everybody writing a component or pattern should be able to implement it. In many cases you will recognize patterns 
+by spotting repetition in user interfaces, solutions to common problems. That's exactly what we are looking for! 
+Describe the problem, describe how it is solved, and create a proposal with our [template]. It doesn't have to be a 
+final version. We are here to help! Every proposal will be reviewed and shaped until it is ready for approval. We 
+believe that first time right doesn't exist, or it is just a lucky shot. Even if you are not sure if the problem you 
+described is a pattern, please submit your idea in order to help others writing even better proposals.
 
 ### Create a new proposal
 
-To create a new proposal fork this repository and create a new markdown file based 
-on the [template] in this repository. You can create a PR with the new file in the
-proposed directory. The file **MUST** be named with the name of your pattern.
+To create a new proposal fork this repository and create a new markdown file based on the [template] in this 
+repository. You can create a PR with the new file in the proposed directory. The file **MUST** be named with the name 
```

Comment:
I think it is better to rename PR to Pull Request, not all the users that are going to cooperate are technical users or github users.